tools/sigclient/pkg/alerts/alerts_loadtest.go

```diff
@@ -77,6 +78,83 @@ func setUpLoggingToFileAndStdOut() {
 	// Set the output of the logger to the multi-writer
 	log.SetOutput(multiWriter)
 
+	return logFile
+}
+
+// tunnel the logs into SigLens
+func startVector(host string, logFile string) (*exec.Cmd, chan error, error) {
+	os.Setenv("LOG_FILE_PATH", logFile)
+	os.Setenv("HOST", host)
+
+	dataDir := filepath.Join("./vector_log_lib/data")
+	err := os.MkdirAll(dataDir, 0764)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to make vector data directory at=%v, err=%v", dataDir, err)
+	}
+	os.Setenv("DATA_DIR", dataDir)
+
+	log.Infof("LOG_FILE_PATH=%s, HOST=%s, DATA_DIR=%s", logFile, host, dataDir)
+
+	configPath := "pkg/alerts/alerts_loadtest_vector.yaml"
+
+	cmd := exec.Command("vector", "-c", configPath)
+
+	// Set the environment for the command
+	cmd.Env = append(os.Environ(),
+		fmt.Sprintf("LOG_FILE_PATH=%s", logFile),
+		fmt.Sprintf("HOST=%s", host),
+		fmt.Sprintf("DATA_DIR=%s", dataDir),
+	)
+
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	err = cmd.Start()
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to start vector: %v", err)
+	}
+
+	errChan := make(chan error)
+
+	go func() {
+		err := cmd.Wait()
+		errChan <- err
+		close(errChan)
+	}()
+
+	return cmd, errChan, nil
+}
+
+func listenForVectorErrors(vectorErrChan chan error, successChan chan bool, runVector int8) {
+	if runVector < 0 {
```

Comment:
nit: converting `runVector` to an enum would help make this more readable. Then you could do something like `if runVector == vectorModeDisabled {`

Reply:
Yes, I have updated the PR with this suggestion.